### PR TITLE
governance: add project management committee(PMC)

### PR DESCRIPTION
**Pull request by winkyao**
Signed-off-by: winkyao <golangwink@gmail.com>

Add the Project Management Committee(PMC) to the README.md. And propose a list of [members](https://github.com/winkyao/community/blob/add-pmc/committee/README.md#members) of PMC.

### CODE REVIEW

File: committee/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# PMC
+
+PMC (Project Management Committee) is to manage and oversee the entire TiKV community, also defines the project values and structure. The PMC consists of Core Maintainers and Organizer Committee Chair.
+
+## Members
```
Comment (Hoverbear) [inline committee/README.md]: This list is missing some of our maintainers, despite it saying it includes our maintainers.

**Comment by Hoverbear** [comment]
@winkyao Can you please show a vote record with our maintainers of this decision? If we are truly a community we cannot have single members solo electing new top level committees.

**Comment by winkyao** [comment]
@Hoverbear Thanks for your suggestion, this page is used to nominate the initial PMC list. Usually, the first list of PMCs is internally elected by TiKV. With the merged member list, we are able to vote for the other PMC members. In addition, the maintainer may not be in the PMC, only the core maintainer will be in the PMC, and this will also have a voting process later.

**Comment by Hoverbear** [comment]
While I'm sure you obtained prior consent from at least a majority of our maintainers (who 'are' TiKV's top governing body in the prior to the PMC: @siddontang, @zhangjinpeng1987, @sunxiaoguang, yourself, amd @BusyJay ), you don't show this and they did not approve this PR.

Do you see how this creates a possible illusion of you acting on your own? We should ensure that future readers know you are acting in good faith and within your duties. :)

**Pull request merged**